composer.json

```
@@ -19,7 +19,7 @@
         "phpunit/phpunit" : "7.0.0",
         "facebook/webdriver" : "dev-master",
         "phan/phan": ">=2.3.0",
-        "phpmd/phpmd": "~2.7"
+        "phpmd/phpmd": "~2.8"
```

```suggestion
        "phpmd/phpmd": "2.8.1"
```
Do you have any feelings about being more strict with this?

---

Why? It seems unnecessary, it would just make "composer update" less likely to get bug fixes. Exact versions if what the lock file is for.